locales/es.json

```diff
@@ -46,170 +46,170 @@
+   "error running onDeploy: ": "error al ejecutar onDeploy: ",
+   " exists but has been set to not deploy": " existe, pero se ha configurado para no desplegar",
+   "couldn't find a valid address for %s has it been deployed?": "no se pudo encontrar una dirección válida para %s ¿ha sido publicado?",
    "executing: ": "ejecutando: ",
    "the transaction was rejected; this usually happens due to a throw or a require": "la transacción fue rechazada; esto generalmente sucede debido a un throw o a un require",
    "no account found at index": "no se encontró una cuenta en index",
    " check the config": " verifique la configuración",
-   "Both \"from\" and \"fromIndex\" are defined for contract": "Ambos \"from\" y \"fromIndex\" están definidos para el contracto",
-   "Using \"from\" as deployer account.": "Usando \"from\" como cuenta de implementación.",
+   "Both \"from\" and \"fromIndex\" are defined for contract": "Ambos \"from\" y \"fromIndex\" están definidos para el contrato",
+   "Using \"from\" as deployer account.": "Usando \"from\" como cuenta de despliegue.",
    "{{linkReference}} is too long": "{{linkReference}} es muy largo",
-   "{{contractName}} needs {{libraryName}} but an address was not found, did you deploy it or configured an address?": "{{contractName}} necesita {{libraryName}} pero no se encontró una dirección, ¿la implementó o configuró una dirección?",
-   "attempted to deploy %s without specifying parameters": "intentado publicar %s sin especificar parámetros",
-   "deployed at": "publicado en",
-   "no contracts found": "no hay contractos encontrados",
+   "{{contractName}} needs {{libraryName}} but an address was not found, did you deploy it or configured an address?": "{{contractName}} necesita de {{libraryName}} pero no se encontró una dirección, ¿fue esta desplegada o configuró una dirección?",
+   "attempted to deploy %s without specifying parameters": "Se intentó desplegar %s sin especificar parámetros",
+   "deployed at": "desplegado en",
+   "no contracts found": "no se encontraron contratos",
    "Blockchain component is disabled in the config": "El componente Blockchain está deshabilitado en la configuración",
    "Couldn't connect to an Ethereum node are you sure it's on?": "No se pudo conectar a un nodo Ethereum, ¿está seguro de que está activado?",
-   "make sure you have an Ethereum node or simulator running. e.g '%s'": "asegurese de tener un nodo o simulador Ethereum en funcionamiento. e.g '%s'",
+   "make sure you have an Ethereum node or simulator running. e.g '%s'": "asegúrese de tener un nodo o simulador Ethereum en ejecución. e.g '%s'",
    "executing": "ejecutando",
    "the transaction was rejected; this usually happens due to a throw or a require, it can also happen due to an invalid operation": "la transacción fue rechazada; esto generalmente ocurre debido a un throw o un require, también puede ocurrir debido a una operación no válida",
```

Comment:
a un **lanzamiento** o un **requerimiento**,...

Reply:
Hi,

I think you're wrong interpreting the texts. Are you aware of the context where the text is being used within the source code ?

Reply:
Hi, apologies, you’re right those are reserved words from js. 😅